refs/schemas/priv/dictionary/statuses/en.statuses.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
@filipblnt In fact, all this is very much work in progress. The question we are askign ourselves (and where we need your input) is whether this format is usfeul for user interfaces as is? Is there maybe a need to have a lable (in english it might be a repeated TERM, or a more user friendly word for the TERM) + a description (more of a tooltip)? Is the format of the JSON map OK?

Best is that you give us instructions for this, and we'll make sure all the dictionaries are made properly.